extensions/tiled-assets/README.md

```diff
@@ -0,0 +1,138 @@
+# Tiled Assets specification
+
+**Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+An extension to allow the specification of tiled assets within STAC Items. With this extension it is possible to allow the description of assets using template references and rules to construct those. This extension is tailored for cases where there are too many assets associated with an Item to be practically listed in the `assets` property. For this reason, the new `asset_templates` property allows to specify template URLs where components can be replaced to get the final URLs to the actual files.
+
+This extension is modelled in close alignment to the [OGC Two Dimensional Tile Matrix Set](http://docs.opengeospatial.org/is/17-083r2/17-083r2.html)
```

Comment:
The extension should tell what other extensions and/or fields may be useful to be used and why (also should have links). For example start_datetime, end_datetime, datacube, ...